data/strings/strings.txt

```diff
@@ -33648,7 +33971,7 @@ fa = دوباره نپرس
     da = Vil du finde populære steder i nærheden?
     nl = Wilt u populaire plaatsen in de omgeving vinden?
     fi = Haluatko löytää suosittuja lähistöllä olevia kohteita
-    fr = Vous voulez trouver des endroits populaires à proximité ?
+    fr = Vous voulez trouver des endroits populaires à proximité ?
```

Comment:
What happened here? Please revert.

Reply:
No problem just lines changed after add new string in fa

Reply:
nothing changed i checked again its right

Reply:
No, it has changed.
The old character was NO-BREAK SPACE (hex codepoint 00A0).
The new character is SPACE (hex codepoint 0020).

The old one was correct. Please revert.
https://en.wikipedia.org/wiki/Non-breaking_space

Reply:
yes i found it 
 fixed

Reply:
Did you push the changes?